docs/projectSetup/iOS/checklist_iOS.md

```diff
@@ -0,0 +1,58 @@
+# iOS Project Setup
+## Git
+Create new repository based on native-ios-repo-template
+
+## Apple Developer
+The Developer Certiciate and Provisioning Profile are required for code signing and distribution.  
+
+[Apple Developer Portal](https://developer.apple.com/account/resources/)
+
+1. Load the NanoGiants GmbH Developer Certificate, and add it into your Keychain.
```

Comment:
Please add a reference where to get it (e.g. Download it via Developer Portal or 1Password). Keep in mind that this might be a Fullstack Developer reading, since they have to deal with this from time to time.

Reply:
You're absolutely right, it's added.